include/zephyr/drivers/entropy.h

```diff
@@ -33,6 +33,9 @@ extern "C" {
  * @typedef entropy_get_entropy_t
  * @brief Callback API to get entropy.
  *
+ * @note This API be thread safe since it can be used
```

Comment:
This API is...?

Reply:
Probably want to just say something like "This call is thread safe". Perhaps "This call is thread safe, to satisfy requirements of the random subsystem."

Reply:
argg... I missed the "This API *needs to*"